people/profit-sharing.md

```diff
@@ -1,16 +1,8 @@
-# Profit sharing
+# Profit Sharing
 
 This document describes how we manage the distribution of the company's profit, on a quarterly basis.
 
-  **Contents**
-  1. [The concept](#the-concept)
-  2. [Past](#past)
-  3. [Present](#present)
-  4. [Future](#future)
-  5. [Donations](#donations)
-
```

Comment:
I kinda like the ToCs at the top ...

Reply:
Then we need to have them everywhere and it adds an additional complexity to the doc. We can have them but I would avoid it. IMO the doc should be short or clear enough that it doesn't need them.